src/Apps/Order2/Routes/Details/Components/Order2DetailsHelpLinks.tsx

```diff
@@ -14,17 +15,18 @@ export const Order2DetailsHelpLinks: React.FC<
   React.PropsWithChildren<Order2DetailsHelpLinksProps>
 > = ({ showInquiry, order, inquiryComponent }) => {
   const orderData = useFragment(fragment, order)
+  const { clickedReadFAQ, clickedAskSpecialist } = useOrder2Tracking()
```

Comment:
This is super minor, but suggestion (here and elsewhere) to not destructure these functions and instead invoke them from their namespace -- 

```
tracking.clickedReadFAQ(...)
```

That way, via the `tracking` keyword, in the future devs can instantly see at a glance (without needing to study the code) that these are tracking functions rather than other kinds of functions.